tests_v2/test_browser_eval_domrect.py

```
@@ -0,0 +1,105 @@
+                f"Expected iframe DOMRect to come from another JavaScript realm: {cross_realm_result}",
+            )
+            _must(
+                isinstance(cross_realm_value.get("rect"), dict),
+                f"Expected cross-realm DOMRect dictionary: {cross_realm_result}",
+            )
+        finally:
+            if surface_id:
+                try:
+                    client._call("surface.close", {"surface_id": surface_id})
+                except cmuxError:
+                    pass
```

Ruff (0.16.0)

[warning] Use `contextlib.suppress(cmuxError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(cmuxError): ...`

(SIM105)